Examples/ObjectiveC/InsetMapExample.m

```diff
@@ -47,7 +47,7 @@ - (void)viewDidLoad {
     self.miniMapView.layer.borderColor = UIColor.blackColor.CGColor;
     self.miniMapView.layer.borderWidth = 1;
     
-    [self.miniMapView setCenterCoordinate:self.mapView.centerCoordinate zoomLevel:self.mapView.zoomLevel animated:NO];
+    [self.miniMapView setCenterCoordinate:self.mapView.centerCoordinate zoomLevel:self.mapView.zoomLevel-2 animated:NO];
```

Comment:
Can you please add a comment as to why we're doing this? (Same for the Swift version)